package.json

```diff
@@ -97,6 +97,27 @@
     "yauzl": "^2.10.0"
   },
   "contributes": {
+    "languages": [
+      {
+        "id": "dtd",
```

Comment:
We also need to add the "onLanguage:dtd" to "activationEvents" since if we open a `.dtd` file first, XML Support won't start.

Reply:
Good catch! Fixed.